src/webapp/azext_webapp/_help.py

```diff
@@ -16,6 +16,15 @@
             az webapp up -n MyUniqueAppName --dryrun \n
             az webapp up -n MyUniqueAppName -l locationName
 """
+helps['webapp remote-connection'] = """
+    type: group
+    short-summary: Create a remote connection using a tcp tunnel to your app
```

Comment:
both have the same help message. How is this different from create?

Reply:
Will this work for function app as well? if not clarify in the help & i also recommend handling this (no support for function app) in the command with a useful error message

Reply:
The first help definition corresponds to the help for the group, while the 'create' command help is the help only for the command itself. It just so happened to be the same for both. Should I change this?